Resources/Prototypes/_StarLight/vote-chances.yml

```
@@ -30,4 +30,7 @@
     memorizeCount: 1
   - id: Extended
     chance: 0.2
+    memorizeCount: 1
+  - id: Revolutionary
+    chance: 0.3
```

```suggestion
    chance: 0.2
```
Way too common, put it the same way as nukies (also holy shit all those vote chances are high, i might need to speak regarding that.)

---

darn it foxxo for to it before me.
Crazy asked me first and I suggested 0.3 in vote weight (because imo revs is a much better mode then nukies)
and even still it is still has cooldown.

---

Current votes fucking sucks in all honest and why its is "better" its should not be common